src/tests/test_filesystem.py

```diff
@@ -432,3 +432,31 @@ async def test_nativefs_dup(selenium, runner):
     close(fd2)
     assert file.read() == "hello_read"
     file.close()
+
+
+@pytest.mark.requires_dynamic_linking
```

Comment:
Why does it require dynamic linking?

Reply:
I forgot to put the same flag in the `ifeq ($(DISABLE_DYLINK), 1)` condition 🙄

Reply:
No problem. I don't think it's bad to have duplicate flags if there are only two.